[0.012s] Invoking command in '/home/kshitij/project5_ws/build/rrt': CMAKE_PREFIX_PATH=/opt/ros/humble:${CMAKE_PREFIX_PATH} /usr/bin/cmake --build /home/kshitij/project5_ws/build/rrt -- -j12 -l12
[0.033s] -- Found ament_cmake: 1.3.8 (/opt/ros/humble/share/ament_cmake/cmake)
[0.152s] -- Found geometry_msgs: 4.2.3 (/opt/ros/humble/share/geometry_msgs/cmake)
[0.179s] -- Found rosidl_generator_c: 3.1.5 (/opt/ros/humble/share/rosidl_generator_c/cmake)
[0.181s] -- Found rosidl_adapter: 3.1.5 (/opt/ros/humble/share/rosidl_adapter/cmake)
[0.188s] -- Found rosidl_generator_cpp: 3.1.5 (/opt/ros/humble/share/rosidl_generator_cpp/cmake)
[0.199s] -- Using all available rosidl_typesupport_c: rosidl_typesupport_fastrtps_c;rosidl_typesupport_introspection_c
[0.215s] -- Using all available rosidl_typesupport_cpp: rosidl_typesupport_fastrtps_cpp;rosidl_typesupport_introspection_cpp
[0.339s] -- Found nav_msgs: 4.2.3 (/opt/ros/humble/share/nav_msgs/cmake)
[0.483s] [31mCMake Error at /opt/ros/humble/share/ament_cmake_python/cmake/ament_python_install_package.cmake:66 (message):
[0.483s]   ament_python_install_package() the Python package folder
[0.483s]   '/home/kshitij/project5_ws/src/rrt/rrt' doesn't contain an '__init__.py'
[0.483s]   file
[0.483s] Call Stack (most recent call first):
[0.483s]   /opt/ros/humble/share/ament_cmake_python/cmake/ament_python_install_package.cmake:39 (_ament_cmake_python_install_package)
[0.483s]   CMakeLists.txt:34 (ament_python_install_package)
[0.483s] 
[0.484s] [0m
[0.484s] -- Configuring incomplete, errors occurred!
[0.484s] See also "/home/kshitij/project5_ws/build/rrt/CMakeFiles/CMakeOutput.log".
[0.488s] gmake: *** [Makefile:240: cmake_check_build_system] Error 1
[0.490s] Invoked command in '/home/kshitij/project5_ws/build/rrt' returned '2': CMAKE_PREFIX_PATH=/opt/ros/humble:${CMAKE_PREFIX_PATH} /usr/bin/cmake --build /home/kshitij/project5_ws/build/rrt -- -j12 -l12
